bundlesize.config.json

```diff
@@ -1,11 +1,7 @@
 {
   "files": [
     {
-      "path": "./packages/**/lib/index.js",
-      "maxSize": "1.5 kB"
-    },
-    {
-      "path": "./packages/**/dist/index.js",
+      "path": "./dist/index.js",
```

Comment:
with the change to `tsup`, are we not outputting individual packages anymore? Or should this be `./packages/**/dist/index.js`?

Reply:
I moved the script to run in the package itself so it's a relative path now. Output is still per package.